pkg/kms/localkms/localkms_writer.go

```diff
@@ -0,0 +1,75 @@
+/*
+ Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package localkms
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/google/tink/go/subtle/random"
+
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+)
+
+// newWriter creates a new instance of local storage key storeWriter in the given store and for masterKeyURI
```

Comment:
Why not return a struct pointer? It is not needed to return a particular interface from a constructor.

also how will this function be used - i see it is not exported from this package.

Reply:
same as reader above

Reply:
set return to struct